gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-apis.yaml

```diff
@@ -1230,6 +1230,26 @@ paths:
                                 $ref: "#/components/schemas/IntegrationIngestionResponse"
                 default:
                     $ref: "#/components/responses/Error"
+    /environments/{envId}/integrations/{integrationId}/_preview:
+        parameters:
+            - $ref: "#/components/parameters/envIdParam"
+            - $ref: "#/components/parameters/integrationIdParam"
+        get:
+            tags:
+                - Integrations
+            summary: Preview APIs to ingest for a specific Integration
+            description: |-
+                Preview APIs to ingest before actual ingestion.
```

Comment:
Could you add information about required permissions here?

Reply:
Sure thing, added below